packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonHeader.tsx

```diff
@@ -371,8 +607,23 @@ const GivingSeasonHeader = ({
                   )}
                 </div>
               )}
-              {rightHeaderItems}
+              <div className={classes.gsRightHeaderItems}>
+                {rightHeaderItems}
+              </div>
             </Toolbar>
+            <div className={classes.gsContent}>
+              <div>
+                <span>Add a heart if you got your 2023 donations in</span>
+              </div>
+              <div className={classes.gsButtons}>
+                <EAButton href="/giving-portal">
+                  Donate to the Donation Election winners
+                </EAButton>
+                <EAButton href="/giving-portal#opportunities">
+                  Explore charities
+                </EAButton>
+              </div>
```

Comment:
FYI [Lizka's post](https://forum.effectivealtruism.org/posts/7D83kwkyaHLQSo6JT/winners-in-the-forum-s-donation-election-2023) about the election winners is out, in case you wanted to link to that here